spec/components/blacklight/gallery/slideshow_preview_component_spec.rb

```diff
@@ -51,6 +51,14 @@
       expect(rendered).to have_selector '.thumbnail img[@src="http://example.com/image.jpg"]'
     end
 
+    it 'renders the correct slide number' do
+      if VIEW_COMPONENT_VERSION < 3
+        expect(rendered).to have_css '[data-slide-to=\"4\"][data-bs-slide-to=\"4\"]'
```

Comment:
Isn't the idea that these numbers should be the same regardless of what version of VC is used?

Reply:
Not in the component.  

It inherits from https://github.com/projectblacklight/blacklight/blob/v8.4.0/app/components/blacklight/document_component.rb#L26C1-L26C72

VIEW_COMPONENT_VERSION = 2, then the COLLECTION_INDEX_OFFSET = 0 
VIEW_COMPONENT_VERSION = 3,  then the COLLECTION_INDEX_OFFSET = 1

VIEW_COMPONENT_VERSION = 2, @count = 0 + 5 (document_counter, which we set in the test)
VIEW_COMPONENT_VERSION = 3, @count = 1 + 5 (document_counter, which we set in the test)

The component removes - 1 so it is right.

Reply:
Okay, so because the input will be different for each version, it's going to result in the same output.